llama_toolchain/core/build_container.sh

```diff
@@ -105,10 +107,10 @@ if [ -n "$LLAMA_MODELS_DIR" ]; then
   mounts="$mounts -v $(readlink -f $LLAMA_MODELS_DIR):$models_mount"
 fi
 set -x
-podman build -t $image_name -f "$TEMP_DIR/Dockerfile" "$REPO_DIR" $mounts
+podman build --network host -t $image_name -f "$TEMP_DIR/Dockerfile" "$REPO_DIR" $mounts
```

Comment:
we cannot assume this for others -- they may not have permissions on their systems to share the host network interface with the container. we can use some ENV var overrides: 

```
DOCKER_BINARY=${DOCKER_BINARY:-docker}
DOCKER_OPTS=${DOCKER_OPTS:-}
```

And then on our machines, we can pass `DOCKER_BINARY=podman DOCKER_OPTS="--network host"`